src/index.html

```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="div-with-input">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            class="age"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+
+        <div class="div-with-input">
+          <label for="birth">Full date of birth:</label>
+          <input
+            id="birth"
+            type="date"
+            name="birth"
+          >
+        </div>
+
+        <div>
+          <label for="accept">I accept the term of the agreement</label>
```

Comment:
Same here, make [for] more descriptive

Reply:
I renamed it for `acceptAgreement`. I renamed silimary other fors.